src/context/src/asset_manager.rs

```diff
@@ -272,3 +316,33 @@ pub enum TextureImpl {
 pub struct Texture {
     texture_impl: TextureImpl,
 }
+
+pub enum ColorBufferImpl {
```

Comment:
Ouch, it sucks to have this kind of enums multiplying. I was hoping for it to be localized and well hidden from anything. It would be great to solve this, but I don't see how...